math/check_prime_sqrtN.cpp

```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+bool isPrime(int n)
+{
+  int sqrtN = sqrt(n);
+  for(int i=2; i <= sqrtN; ++i)
+  {
+    if(n % i == 0)
+    {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+int main()
+{
+  int n;
+  cin >> n;
+
+  assert(n <= 1000000000);// Check if the given integer is too large
```

Comment:
This line is useless because the ```sqrt(1e+09)``` is  ca. ```31622```. And this isn't a problem of time by a modern computer.